src/assets/scss/tokens/typography.scss

```diff
@@ -1,20 +1,18 @@
 /* @link https://utopia.fyi/type/calculator?c=320,16,1.125,1280,16,1.25,6,2,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l */
 
-:root {
```

Comment:
I believe this needs to stay where it is due to the @media(screen) block below changing `--fluid-screen`. Otherwise, the order of overrides is backwards.

Reply:
This was a miss from my side. Will take time to iterate upon this and fix it